x-pack/platform/plugins/shared/onechat/public/application/components/conversations/conversation.tsx

```diff
@@ -78,9 +76,10 @@ export const Conversation: React.FC<{}> = () => {
     min-height: 0;
   `;
 
+  // TODO: Add custom mask for overflow scroll top and bottom
```

Comment:
Will do this in a future PR, this is not priority for now when we have other things to do. Adding a mask means padding gets added at the top and bottom of the conversation scroll container and this messes with our dynamic height calculation for the height of the round. It needs more thought.